string fileName
---
string[] varNames
string[] varTypes
